internal/manager/config.go

```diff
@@ -152,7 +152,7 @@ func (c *Config) FlagSet() *pflag.FlagSet {
 	flagSet.Float32Var(&c.ProxyTimeoutSeconds, "proxy-timeout-seconds", dataplane.DefaultTimeoutSeconds,
 		"Sets the timeout (in seconds) for all requests to Kong's Admin API.",
 	)
-	flagSet.StringVar(&c.KongCustomEntitiesSecret, "kong-custom-entities-secret", "", `A Secret containing custom entities for DB-less mode, in "namespace/name" format`)
+	flagSet.StringVar(&c.KongCustomEntitiesSecret, "kong-custom-entities-secret", "", `No longer has any effect and will be removed in a later release (see github issue #3278)`)
```

Comment:
```suggestion
	flagSet.StringVar(&c.KongCustomEntitiesSecret, "kong-custom-entities-secret", "", `WARNING: Does not work. It's a known issue tracked in a GitHub issue #3278.`)
```

Maybe this way, to not make the decision about removing in the next release before actually making the decision?